src/components/Menu/menu.scss

```diff
@@ -350,7 +351,8 @@
         z-index: 1030;
         padding: 2vw;
         top: 0vh;
-        height:12vw;
+        height:19.2vw;
+        line-height: 50px;
```

Comment:
This causes some issues on small mobile when the locations item is expanded (gotta fit all the items on the screen)

Reply:
Ok. I'll come up with a different solution for it.